sssd_test_framework/topology.py

```diff
@@ -74,6 +74,17 @@ def test_ldap(client: Client, ldap: LDAP):
     .. topology-mark:: KnownTopology.IPA
     """
 
+    IPA2Clients = SSSDTopologyMark(
```

Comment:
Looking at the other proposed topologies. It will be better to spell everything out. 

``` IPATwoClients ```

We have a pending topology, ``` SambaTwoDCOneGC ``` then using numbers it will look like  ``` Samba2DC1GC ``` which is a little confusing. Let's standardize on spelling everything out when it comes to topologies.